lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -407,6 +407,12 @@ def _prepare_environment(  # noqa: C901
                     for azure_cap in location_info.capabilities:
                         if azure_cap.vm_size == node_runbook.vm_size:
                             predefined_cost += azure_cap.estimated_cost
+                            # set a min value for nic_count
+                            # temp work around for an azure python sdk bug
```

Comment:
remove the word `temp` since we don't have eta about when this "temp" workaround can be removed. We may need to keep it forever.